blocks/aem-asset-selector/aem-asset-selector-util.js

```diff
@@ -4,9 +4,11 @@ const IMS_ENV_STAGE = 'stg1';
 const IMS_ENV_PROD = 'prod';
 const API_KEY = 'franklin';
 const WEB_TOOLS = 'https://master-sacred-dove.ngrok-free.app';
+// TODO: change this to Asset Link IMS client ID
```

Comment:
If we can host the selector on a single domain, we can use a single client ID for everyone. We probably wouldn't be able to use a single client ID if the selector is hosted on every customer's Franklin site.